### ISSUE TITLE
Reviewer 2 Comment 4

### ISSUE THREAD

**Issue by ramess101** [ramess101/JCED_FOMMS_Manuscript#28]
@mrshirts @jpotoff @msoroush 

> Eq. 17: internal energies for vapor phases are suffering strong finite size scaling effects in GCMC simulations with low number / vanishing number of molecules. Among all quantities considered in this work, it is probably the least size-robust quantity. The enthalpy of vaporization determined from eq. 17 therefore suffers from substantial finite size effects. Robust values for enthalpies of vaporization that more closely resemble infinite size systems are determined from the  Clausius-Clapeyron equation (as stated by Weidler and Gross, 2016, see above). The derivative dln(p_sat)/d(beta) is a native quantity in the spirit of histogram reweighing.

Here is Eq 17:

![image](https://user-images.githubusercontent.com/23408516/53427753-c42abb80-39a6-11e9-807e-333961cf9a3f.png)

Here is what Weidler and Gross stated:

![image](https://user-images.githubusercontent.com/23408516/53427787-d60c5e80-39a6-11e9-9b1b-44d9cb109359.png)

I don't think we want to change how we compute DeltaHv for two reasons. First, this will mess up the validation comparison with GCMC-HR (Figures 1 and 3). Second, the individualized optimizations (Figure 2) will be inconsistent with the original MiPPE parameterization (for branched alkanes only since alkynes do not include DeltaHv). We could modify the approach for the cyclohexane optimization, but I think this will only confuse the reader to have two DeltaHv approaches. @jpotoff @msoroush what are your thoughts?

**Comment by jpotoff** [comment]
@ramess101 I'm not eager to change how we calculate dHv for this paper.  What they propose may be better, but as you mentioned, it's inconsistent with how we calculated dHv in the past, and it's going to throw off the validation of MBAR and confuse people.   The paper is not about system size effects (we are writing another paper on that, btw), and the data show reasonably small system size effects except in the near critical region, anyway.  Using this other method is not going to produce substantially different results, nor would it change the optimal parameters, since the scoring function is designed to focus primarily on liquid density and vapor pressure.

That said, we could try it on the cyclohexane data and see what we get.   Not opposed to using this method in the future, but I think it's not a good idea to change it for this paper.

**Comment by ramess101** [comment]
@jpotoff 

> I'm not eager to change how we calculate dHv for this paper. What they propose may be better, but as you mentioned, it's inconsistent with how we calculated dHv in the past, and it's going to throw off the validation of MBAR and confuse people.

I agree.

> The paper is not about system size effects (we are writing another paper on that, btw), and the data show reasonably small system size effects except in the near critical region, anyway.

To satisfy this reviewer we can also mention the finite-size effects investigation we performed for cyclohexane.

> Using this other method is not going to produce substantially different results, nor would it change the optimal parameters, since the scoring function is designed to focus primarily on liquid density and vapor pressure.

For example, we still got essentially the same optimal as the TAMie force field.

> That said, we could try it on the cyclohexane data and see what we get. Not opposed to using this method in the future, but I think it's not a good idea to change it for this paper.

I think we leave this approach to future studies if we find it to be necessary.

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

My response:

![image](https://user-images.githubusercontent.com/23408516/53756172-c6dd5300-3e75-11e9-9abf-17de0f05d51a.png)

![image](https://user-images.githubusercontent.com/23408516/53756224-e3798b00-3e75-11e9-9a2c-d4a785beffdf.png)

![image](https://user-images.githubusercontent.com/23408516/53756285-01df8680-3e76-11e9-92f1-f61c10820909.png)

**Comment by msoroush** [comment]
@ramess101 @jpotoff 
Here are the results for calculating the dHV using GCMC + HR (dHv, blue), Weidler and Gross (dHv2, red), and GEMC (green). There are good agreements in high temperature for all 3 methods, but at low temperature, Weidler and Gross deviates by ~0.6 kcal/mol. I attached the results for you. 

![heatofvap](https://user-images.githubusercontent.com/13317444/54770239-33c44d00-4bd9-11e9-8218-ccb82e27a6d8.png)
[out_average_GEMC.txt](https://github.com/ramess101/JCED_FOMMS_Manuscript/files/2993131/out_average_GEMC.txt)
[out_butane_40A.txt](https://github.com/ramess101/JCED_FOMMS_Manuscript/files/2993132/out_butane_40A.txt)

**Comment by ramess101** [comment]
@msoroush 

Very interesting. This seems to suggest that our approach is actually better than Weidler and Gross against finite-size effects at low T. I guess, is it possible GEMC has finite-size effects for such low densities in the vapor phase at low T?

**Comment by msoroush** [comment]
@ramess101  I have around 160 molecule in my gas phase. Usually GEMC shows finite size effect near critical points. Have you tried to apply the Weidler and Gross for cyclohexane results?

**Comment by ramess101** [comment]
@msoroush 

Yeah, I have never seen traditional finite-size effects in GEMC at low T. But if N = 0 in the vapor phase it could be a similar problem to GCMC with poor averaging. This could also come back to the discussion we had previously about <N>/<V> vs <N/V> and <U>/<N> vs <U>/<N>. Where this distinction doesn't impact the Weidler and Gross analysis.

I haven't applied their approach.

**Comment by jpotoff** [comment]
Just following up on this because we have some new data.  If one calculates the d ln p/d (1/T) at each temperature of interest, no significant difference is observed between the two methods.
